src/sentry_plugins/heroku/plugin.py

```diff
@@ -36,7 +38,7 @@ def handle(self, request):
                     "email": email,
                 },
             )
-        self.finish_release(version=request.POST["head_long"], url=request.POST["url"], owner=user)
+            self.finish_release(version=request.POST["head_long"], url=request.POST["url"])
```

Comment:
@NisanthanNanthakumar I am a bit confused why this fixes anything. The error in Sentry is a key error on `head_long`, yea? Wouldn't we still have the same problem?

Reply:
@scefali No you're right. My brain's not working at all. I think I got attracted to `owner.missing` and just focused on that. It seems that the people affected by this are not configuring their webhooks properly in heroku to POST the correct information. It seems that they are using [this](https://devcenter.heroku.com/articles/webhook-events#api-release) when they should be using [this](https://devcenter.heroku.com/articles/deploy-hooks#http-post-hook) instead. I think this ticket involves updating docs.

Reply:
I don't think we need to update the docs bc in Sentry, we give users the exact command to paste into terminal that sets up the webhook correctly. It's on them for rolling their own.

Reply:
@NisanthanNanthakumar Ok, I think we need to return a 400 if `head_long` is missing in the payload.

Reply:
When `head_long` is missing, it will now set `version` to `None` which is not a valid Release version. This will raise a `HookValidationError`.